### DISCUSSION TITLE
VMWare: Private and Insecure Local Image Registries

### DISCUSSION THREAD

**Discussion by trebonian** [bottlerocket-os/bottlerocket#1644] [Q&A]
How do you go about setting containerd repository mirrors and configs on a VMWare network that isn't connect to a public cloud image repository?

I have read [#1227](https://github.com/bottlerocket-os/bottlerocket/issues/1227) and would also like to include insecure local repositories. On a VMWare network - the security would be provided by an isolated physical network....

**Comment by etungsten** [comment]
Hi @trebonian, there is an open PR out currently (https://github.com/bottlerocket-os/bottlerocket/pull/1629) that would allow you to configure registry mirrors for containerd. Please let us know if that would fit your use-case!

**Reply by trebonian** [reply]
I think it will - if it allows insecure mirrors...(which your PR demonstrates)..

Any idea when #1629 will be merged in?

**Reply by etungsten** [reply]
Unfortunately I don't have a specific date at the moment. But once it's merged it should be in our next official release. Releases have typically been six weeks apart, aside from urgent issues.

**Comment by trebonian** [comment]
I am also building from the development branch, so I should be able to access it once it's merged.

Will it be merged sooner?

**Reply by zmrow** [reply]
It really depends on the PR.  Provided that the PR can make it through review before the release, we will merge it sooner.

**Comment by etungsten** [comment]
Hi trebonian, #1629 has been merged and released in Bottlerocket [v1.2.0 ](https://github.com/bottlerocket-os/bottlerocket/releases/tag/v1.2.0).  Please try it out and let us know if it works for your use-case.

**Comment by trebonian** [comment]
I will give it a try. Thanks
